### DISCUSSION TITLE
dispatchTransaction function of Prosemirror in Tiptap

### DISCUSSION THREAD

**Discussion by ankit9015** [ueberdosis/tiptap#3794] [General Questions] [answered]
Hello, how can we call dispatchTransaction function of Prosemirror in tiptap? Couldn't find anything related to it in tiptap docs

Reference: https://prosemirror.net/docs/ref/#view.DirectEditorProps.dispatchTransaction

**Comment by bdbch** [comment, marked as answer]
Hey, Tiptap now supports https://tiptap.dev/docs/editor/extensions/custom-extensions/create-new/extension#dispatchtransaction
